actions/bind

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Configuration helper for BIND server.
+"""
+
+import argparse
+
+from plinth import action_utils
+
+
+CONFIG_FILE = '/etc/bind/named.conf.options'
+
+value1 = 'acl goodclients { \n   localhost;\n};\n'
+value2 = '        recursion yes;\n           allow-query { goodclients; };\n\n'
+value3 = '	// 	8.8.8.8;\n	// 	8.8.4.4;\n'
+value4 = '	//forward only;\n'
```

Comment:
Should this be "forward first" instead of "forward only"?

Also, it seems like both forwarding and DNSSEC would be popular features. Should they be enabled by default?